app/javascript/packages/countdown/countdown-alert-element.spec.ts

```diff
@@ -5,9 +5,14 @@ import './countdown-element';
 describe('CountdownAlertElement', () => {
   const sandbox = useSandbox({ useFakeTimers: true });
 
-  function createElement({ showAtRemaining }: { showAtRemaining?: number } = {}) {
+  function createElement({
+    showAtRemaining,
+    redirectURL,
+  }: { showAtRemaining?: number; redirectURL?: string } = {}) {
     document.body.innerHTML = `
-      <lg-countdown-alert${showAtRemaining ? ` show-at-remaining="${showAtRemaining}"` : ''}>
+      <lg-countdown-alert
+        ${showAtRemaining ? ` show-at-remaining="${showAtRemaining}"` : ''}
+        ${redirectURL ? `redirect-url ="${redirectURL}"` : ''}>
```

Comment:
Couple small details:

* Excess space before the `=`
* I think we should be consistent with the leading space in the template string. Technically we don't need it anymore since the tabbing would suffice

```suggestion
        ${showAtRemaining ? `show-at-remaining="${showAtRemaining}"` : ''}
        ${redirectURL ? `redirect-url="${redirectURL}"` : ''}>
```